blog/_posts/2016-10-19-scalajs-bundler.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: blog
+post-type: blog
+by: Julien Richard-Foy
+title: "scalajs-bundler has been released"
+---
+
+This week we have released a first version of [`scalajs-bundler`](https://scalacenter.github.io/scalajs-bundler), a tool aiming to provide an integrated developer experience for
+[Scala.js](https://www.scala-js.org/) projects that use [npm](https://www.npmjs.com/) packages.
+
+## Motivation
+
+npm is the most popular JavaScript package registry. How can Scala.js applications benefit from
+the many libraries published on npm? First, this requires to resolve these libraries, as well as
+their transitive dependencies, from the npm registry and to download them. Then, the artifacts
+need to be somehow “linked” with the Scala.js code. This latter step is more subtle than it seems
```

Comment:
... need to be ~~somehow~~ *linked* with the Scala.js code. This last step is subtler than it seems -- the linking process varies according to the target execution environment.